CHANGELOG.md

```diff
@@ -6,6 +6,13 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic
 Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.7.0]
+
+### Changed
+
+- Use GATs for `Encoder` trait to allow encoding of borrowed data.
+  See [PR 9](https://github.com/mxinden/asynchronous-codec/pull/9).
```

Comment:
What do you think of including the diff to the `LengthCodec` below here to help folks upgrade to the new version?

Reply:
Are the docs and the link to the PR not enough?

Reply:
It reduces one step for users.I think it might safe them some time. Not a strong preference.

Reply:
Would prefer to just merge this to be honest :)